README.md

```diff
@@ -2,68 +2,117 @@
+It is kept under our `script` directory.
+
+After cloning the repo and navigating into the cloned directory, run the init script
+in your shell. To do so in bash, for example, invoke:
 
-### Clone repo and run init script 
-We have an init script that installs a linter (rubocop), as well as sets up a 
-precommit hook to have the linter automatically run whenever you make a commit.
-Clone this repo, navigate into the cloned directory, and run 
 ```
-./script/init-setup.sh
+bash script/init-setup.sh
 ```
 
-### Running in a Docker Container
-To launch the app, simply navigate into the cloned directory and run
+### Running the backend with Docker
+
+To begin, make sure that Docker is running on your system. Once it is, invoke
+
 ```
 docker-compose up
 ```
 
-This will launch two containers: a rails app, and a postgres database. Access
-the rails app by navigating to `http://localhost:3000`
+from the cloned directory.
+
+This will launch two containers: a rails app, and a postgres database. You can 
+access the rails app from your browser by navigating to `http://localhost:3000`. 
+(Bare in mind that in tapp, rails is meant only to serve the API, not views.)
+
+If you already have an existing image running on your machine that you would
+like to rebuild, run
 
-If you have an existing image on your machine that needs to be updated, run
 ```
 docker-compose up --build
 ```
 
-To view the STDOUT from a docker container of a running server, you can do
-`docker-compose logs -tf <image>`, like: `docker-compose logs -tf tapp`.
+To view the STDOUT from a docker container of a running server, you can invoke
+
+```docker-compose logs -tf <image>```
+
+### Navigating into the containers from the command line
+
+Currently, we define two services under docker-compose: tapp and db. If you would
+like to interact with either container from the command line, you can do so by 
+invoking:
 
-To access the dockerized app, simply run
 ```
-docker-compose exec tapp sh
+docker-compose exec [service] sh
 ```
 
-### Initializing DB
-To create and setup your local development database, simply navigate into the
-rails container and run the following rake task. It will create your local database,
-run all migrations, and populate the db with seed data. 
+To interact with the rails app, invoke `docker-compose exec tapp sh` and to
+interact with the database, invoke `docker-compose exec [db] sh`.
+
+### Initializing the Database
+
+To both create and setup your local development database, navigate into the
+rails container and run: 
 
 ```
 rake db:setup
 ```
 
-Once your DB is setup, you can have SQL access to it through your docker container by running
+This will create your local database, run all migrations and populate the DB 
+with seed data. 
+
+Once your DB is setup, you can gain SQL access to it by first navigating into 
+the DB container, then invoking:
+
 ```
 $ psql tapp_development tapp
 ```
 
+## Running the frontend
```

Comment:
I know this is a hard question.. but what is the "front end" in this usage?
I reckon it's how we serve the javascript of our app -- but I don't understand how we might start up the "front end" in production differently than from dev. 

Maybe I'm below the threshold of a reasonable knowledgeable reader?

Reply:
Given how most people tend to use the term these days I don't think it will pose a problem. That said, the "frontend" here is how we serve the collection of *both* html, stylings (i.e. css) and javascript (note that rails does not serve any of these things). In other words, it's the presentational aspect of the project. 

I specified in the document that this is separate from the "backend" (the logical/state aspect of the app). In retrospect, this distinction might deserve its own section altogether.

Reply:
I think to clarify this point, maybe you can change "frontend" to "react frontend app" since the frontend is sort of its own app, separate from the Rails API server?

Reply:
Sure, sounds good to me. I'll add a commit addressing that.